bin/cmd.js

```diff
@@ -404,18 +415,18 @@ function runDownload (torrentId) {
           return fatalError(err)
         }
         playerArgs.vlc[0] = vlcCmd
-        openPlayer(playerArgs.vlc.concat(JSON.stringify(href)))
+        argv.playlist ? openPlayer(playerArgs.vlc.concat(all_hrefs)) : openPlayer(playerArgs.vlc.concat(JSON.stringify(href)))
```

Comment:
don't they need to be JSON stringified?

Reply:
Each href is `JSON.stringified` see line 397.